pkg/util/goschedstats/runtime_go1.21.go

```diff
@@ -0,0 +1,200 @@
+// Copyright 2023 The Cockroach Authors.
```

Comment:
Isn't this file identical to the 1.20 file? If it is, just change the `!go1.21` to `!go1.22` on that file instead. (and modify the comment on line 11 to say that it was cross-checked against go1.20 and go1.21).